nexus-migration/maven-central-publish/README.md

```
@@ -0,0 +1,147 @@
+  - Add the `settings.xml` managed file:
+    - File: *(select your stored settings.xml by ID)*
+    - Variable: `CENTRAL_SETTINGS_XML`
+
+- **✅ Inject passwords / credentials** *(Credentials Binding)*
+  - Add a **Username and password (separated)** binding:
+    - Credential: `maven-central-token`
+    - Username variable: `CENTRAL_TOKEN_USERNAME`
+    - Password variable: `CENTRAL_TOKEN_PASSWORD`
+
+- **✅ Inject environment variables** *(EnvInject)* — add to *Properties Content*:
+  ```
```

markdownlint-cli2 (0.22.1)

[warning] Fenced code blocks should have a language specified

(MD040, fenced-code-language)